src/app.rs

```diff
@@ -826,6 +849,28 @@ fn render<T: Layout>(
     });
 
     txn.set_root_pipeline(window.internal.pipeline_id);
+
+    let keys = window.scroll_states.0.keys().map(|k| *k).collect::<Vec<_>>();
+    for (key, value) in window.scroll_states.0.iter_mut() {
+        let (x, y) = value.get();
+        txn.scroll_node_with_id(LayoutPoint::new(x, y), *key, ScrollClamping::ToContentBounds);
+    }
+
+    // if let Some(cursor_position) = window.state.mouse_state.cursor_pos {

```

Comment:
Remove this comment block if it's not needed anymore, it's in the git history anyway, if absolutely necessary.